package.json

```diff
@@ -19,7 +22,11 @@
     "url": "https://github.com/ethomson/probot-serverless-azurefunctions/issues"
   },
   "homepage": "https://github.com/ethomson/probot-serverless-azurefunctions#readme",
-  "dependencies": {
-    "probot": "^7.4.0"
+  "devDependencies": {
+    "jest": "^26.6.1",
+    "probot": "^10.9.5"
+  },
+  "peerDependencies": {
+    "probot": "^10.9.5"
```

Comment:
Why did you move `probot` into dev dependencies?

I would also pin probots version, because you use internal APIs, I cannot guarantue that they won't change in future:

```js
const { resolve } = require('probot/lib/helpers/resolve-app-function')
const { findPrivateKey } = require('probot/lib/helpers/get-private-key')
```

We might turn `findPrivateKey` into an external package, as it might be useful outside of Probot, and we want to make it easy to "eject" probot once you outgrow its scope.

Not sure about the `resolve` function, I'll have to look into that.

Reply:
Good catch, not sure how I managed that one. Updated